installers-conda/build_installers.py

```diff
@@ -135,10 +135,10 @@
 indent4 = partial(indent, prefix="    ")
 
 base_specs = {
-    "python": f"={PY_VER}",
+    "python": "=3.11.9",
```

Comment:
We could soften the pin on Python for the base environment, as you suggest. I chose a hard pin for the base environment because I thought it prudent to update the base environment as little and infrequently as possible and the benefit of a Python update in the base environment is extremely small. However, I don't feel too strongly one wya or the other. If you'd like to soften pin, I can do that.

Reply:
Note that `PY_VER` is effectively pinned by the version set in the workflow. This must be explicitly updated. So by softening the pin on the base environment here, it will be _effectively_ softer than the runtime environment.

Reply:
Ok, sorry. I thought this pin was for the runtime env, not the base one.

I agree with you that it's better to pin Python to a specific version in that case.